refl1d/reflectivity.py

```diff
@@ -115,14 +117,13 @@ def reflectivity_amplitude(kz=None,
     else:
         irho = _dense(irho, 'd')
 
-
-    #print(irho.shape, irho[:,0], irho[:,-1])
+    # print(irho.shape, irho[:,0], irho[:,-1])
     irho[irho < 0] = 0.
-    #print depth.shape, rho.shape, irho.shape, sigma.shape
-    #print depth.dtype, rho.dtype, irho.dtype, sigma.dtype
+    # print depth.shape, rho.shape, irho.shape, sigma.shape
+    # print depth.dtype, rho.dtype, irho.dtype, sigma.dtype
     r = np.empty(kz.shape, 'D')
-    #print "amplitude", depth, rho, kz, rho_index
-    #print depth.shape, sigma.shape, rho.shape, irho.shape, kz.shape
+    # print "amplitude", depth, rho, kz, rho_index
+    # print depth.shape, sigma.shape, rho.shape, irho.shape, kz.shape
```

Comment:
Personal style: I use no spaces when commenting out code vs. one space when writing comments about code. *Black* will of course ignore me and put a space in there.

Reply:
The auto code formatter in VS code puts in the space... and if we are going to try to follow pep8 I think it's too much work to try to fight both battles.

Reply:
My solution would be to avoid the vscode formatter since it has already demonstrated in other contexts that it doesn't know how to format python code.

Reply:
If you can suggest an alternative formatter to use in vscode I would be happy to use it, but when formatting thousands of lines we can't really do it by hand.  I just checked and what I'm using now is "autopep8".  I also have "black" and "yapf".  
I think some of the earlier issues I had with it were user error (on my part).  If you can find errors in the many "pep8" commits that I just pushed I will adjust my settings.

Reply:
My earlier point was that it is unsustainable to try to maintain a style that is not automatically respected by autoformatters (or automatically enforced by them).  If we are going to enforce style, we have to rely on the tools.

Reply:
The main tool I've used on other projects is lint, with periodic manual cleanup of lint errors. We had a discussion on the sasmodels repo about a tool to check whether a commit increased the lint count and flag those that do. Seems mostly doable, but we didn't act on it.

Regarding auto-formatters, PEP 8 was intended to be prescriptive not descriptive. Code "readability" trumps formatting consistency. When vscode breaks a line in the middle of an expression rather than breaking at a comma that hurts readability. If you would like to use a code formatter please look for one that knows how to format code.

Reply:
I see settings lets you choose between black, autopep8 and yapf. I haven't tried any of them.